examples/dev/Dashboard.html

```diff
@@ -35,6 +35,11 @@
         color: purple;
       }
     </style>
+
+    <!-- polyfills for IE11 -->
+    <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script>
```

Comment:
1. I don’t think both promise files are needed, we used to recommend just `es6-promise.auto.min.js`, and I think it worked in our tests: https://github.com/transloadit/uppy/pull/1576/files. I could be wrong.
2. We have these polyfills included in the website examples, like https://uppy.io/examples/dashboard/. Maybe that’s enough for testing, and we could keep `dev` smaller/faster? On the one hand, it’s not too big of an addition, on the other it’s external resources just for IE testing.

Reply:
1. Nice, I'll check and change it if we decide to leave scripts.
2. Running `4000` is too heavy for development, on every update it reloads multiple times.
We could try making <script>s async if you are worried about the speed, or put them somewhere a little lower down the page.
Maybe I'm biased because I had to test on IE11 so much, do we not usually do it, did you not have to be adding these scripts yourself?

Reply:
2. It is too heavy for development, totally, that’s why `npm run dev` exists. I thought we could do quick IE checks with the website, and actual dev work in the dev example. But I agree, I did have to add polyfills too, if we are struggling with this, you are right, we should just add them. 

Thought: should we add them to the bundle maybe, like this https://github.com/transloadit/uppy/blob/a52e3f90b433621260933ab641e091c536a72d05/website/src/examples/dashboard/app.es6#L1-L2 maybe possibly commented out, so we can uncomment quickly for IE? Then it works for offline dev.

But this is nitpicking.

> try making <script>s async

I think that would mean Uppy is loaded before polyfills and result in errors.

Reply:
+1 for adding those requires to the dev bundle. I don't think they need to be commented out, as the modules are small and incur at most a few ms one-time cost at startup

Reply:
Sounds good, that also saves us from needing to import these scripts for both /dashboard.html and /dragdrop.html.